config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-presubmits.yaml.tpl

```diff
@@ -135,7 +135,6 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api{{ if eq $.branch "main" | not -}}{{ TrimPrefix $.branch "release" }}{{- end }}
       testgrid-tab-name: capi-pr-test-mink8s-{{ ReplaceAll $.branch "." "-" }}
```

Comment:
A few more occurences in the periodics template

Reply:
didn't drop them on purpose because they were not only for outdated versions but fine to drop them 👍